### PR TITLE
Remove custom port limit for alexa account linking

### LINKED ISSUE #29095: Alexa Setup
### Feedback

Authorization URI: https://[YOUR HOME ASSISTANT URL]/auth/authorize

Access Token URI: https://[YOUR HOME ASSISTANT URL]/auth/token

Documentation states that custom ports cannot be used.
That's (no longer?) correct.

You can use  https://[YOUR HOME ASSISTANT URL]:1443/auth/authorize and Access Token URI: https://[YOUR HOME ASSISTANT URL]:1443/auth/token to specify a specific port. 

The only important thing is, that it needs to be SSL with a valid SSL Ceritificate.

### URL

https://www.home-assistant.io/integrations/alexa.smart_home/

### Version

2023.9.3

### Additional information

,

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Remove custom port limit for alexa account linking

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #29095

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/alexa.smart_home.markdown [hunk 1]
```diff
@@ -229,10 +229,18 @@ Alexa needs to link your Amazon account to your Home Assistant account. Therefor
 - Find the skill you just created, click `Edit` link in the `Actions` column.
 - Click `ACCOUNT LINKING` in the left navigation bar of build page
 - Do not turn on the "Allow users to link their account to your skill from within your application or website" switch. This will require a Redirect URI, which won't work.
-- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]`. Alexa account linking does not work with a non-standard port. You must use port 443. Use your firewall to forward, if needed. Using a `:1234` or a similar port number will not work.
+- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]`. For Alexa account linking by default the standard port 443 is used. Use your firewall to forward this, if needed:
```
Comment (c0ffeeca7) [inline source/_integrations/alexa.smart_home.markdown]:
```suggestion
- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]`. For Alexa account linking, by default, the standard port 443 is used. Use your firewall to forward this, if needed:
```

File: source/_integrations/alexa.smart_home.markdown [hunk 2]
```diff
@@ -229,10 +229,18 @@ Alexa needs to link your Amazon account to your Home Assistant account. Therefor
...
   - `Authorization URI`: `https://[YOUR HOME ASSISTANT URL]/auth/authorize`
   - `Access Token URI`: `https://[YOUR HOME ASSISTANT URL]/auth/token`
-    - Note: you must use a valid/trusted SSL Certificate for account linking to work
+
+    It is also possible to use a different port by appending `:1443` or a similar port number, make sure your firewall is forwarding the correct port:
+  - `Authorization URI`: `https://[YOUR HOME ASSISTANT URL]:1443/auth/authorize`
+  - `Access Token URI`: `https://[YOUR HOME ASSISTANT URL]:1433/auth/token`
+
+<div class="note">
+    Note: you must use a valid/trusted SSL Certificate for account linking to work. Self signed certificates will not work, but you can use a free Lets Encrypt Certificate.
```
Comment (c0ffeeca7) [inline source/_integrations/alexa.smart_home.markdown]:
```suggestion
    Note: you must use a valid/trusted SSL certificate for account linking to work. Self signed certificates will not work, but you can use a free Let's Encrypt certificate.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @jbouwh :+1:

**Comment by jbouwh** [comment]
Thnx!

**Pull request merged**